wp-modules/api-data/api-data.php

```diff
@@ -122,6 +134,26 @@ function delete_pattern( $request ) {
 		: new WP_REST_Response( $is_success, 400 );
 }
 
+/**
+ * Updates the list of sites that should not show environment notifications.
+ *
+ * @param WP_REST_Request $request Full data about the request.
+ * @return WP_REST_Response
+ */
+function update_dismissed_sites( $request ) {
+	$dismissed_sites = array_merge( get_dismissed_sites(), (array) get_current_blog_id() );
```

Comment:
Storing `$dismissed_sites` as an array here so dismissal will still work as expected for multisite.

If we stored just the site id, a multisite admin would be overwriting any previously dismissed sites each time they dismissed the notice (as opposed to adding the dismissal to an array of previously dismissed sites).

Reply:
Nice

Reply:
Also calling out usage of `get_current_blog_id()` as the site id.

I think it makes sense as a value that should not change, regardless of if the name of a site changes. But please check me if there is a more appropriate value to use!